src/bootstrap/choices.tpl.html

```diff
@@ -1,11 +1,12 @@
 <ul class="ui-select-choices ui-select-choices-content ui-select-dropdown dropdown-menu"
-    ng-show="$select.open && $select.items.length > 0">
-  <li class="ui-select-choices-group" id="ui-select-choices-{{ $select.generatedId }}" >
+  ng-show="$select.open && $select.items.length > 0">
+  <li class="ui-select-choices-group" id="ui-select-choices-{{ $select.generatedId }}">
     <div class="divider" ng-show="$select.isGrouped && $index > 0"></div>
     <div ng-show="$select.isGrouped" class="ui-select-choices-group-label dropdown-header" ng-bind="$group.name"></div>
     <div ng-attr-id="ui-select-choices-row-{{ $select.generatedId }}-{{$index}}" class="ui-select-choices-row"
-    ng-class="{active: $select.isActive(this), disabled: $select.isDisabled(this)}" role="option">
-      <span class="ui-select-choices-row-inner"></span>
+      ng-class="{active: $select.isActive(this), disabled: $select.isDisabled(this)}" role="option">
+      <span class="ui-select-choices-row-inner" ng-if="!value.$null"></span>
```

Comment:
Where does `value` come from?

Reply:
Yeah, fixed that with some... annoying code. `this[$select.itemProperty].$$null` surprisingly works.